tests/models/mixtral/test_modeling_mixtral.py

```diff
@@ -493,6 +492,97 @@ def test_load_balancing_loss(self):
         # This is to mimic torch.testing.assert_not_close
         self.assertNotAlmostEqual(include_padding_result.aux_loss.item(), result.aux_loss.item())
 
+    # Ignore copy
```

Comment:
The code pattern in this test is weird, and we don't rely much on mixtral's tests (it's expensive to run) -- I'd remove this test

Reply:
Removed the tests for mixtral!